Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js

```diff
@@ -340,6 +344,15 @@ WI.DOMNodeStyles = class DOMNodeStyles extends WI.Object
         target.CSSAgent.getInlineStylesForNode.invoke({nodeId: this._node.id}, wrap.call(this, fetchedInlineStyles, fetchedInlineStylesPromise));
         target.CSSAgent.getComputedStyleForNode.invoke({nodeId: this._node.id}, wrap.call(this, fetchedComputedStyle, fetchedComputedStylesPromise));
 
+
+        if (InspectorBackend.hasCommand("CSS.getCustomFonts")) {
+            target.CSSAgent.getCustomFonts((error, fontFamilyNames) =>{
+                if (error)
+                    return;
+                this._customFonts = fontFamilyNames;
```

Comment:
i think this should dispatch some sort of `WI.CSSManager.Event.FontsChanged` so that listeners (e.g. `WI.FontDetailsPanel`) can listen to it and update accordingly